src/content/cloud/okteto-pipeline.mdx

```diff
@@ -7,6 +7,8 @@ id: okteto-pipeline
 
 Okteto Pipelines allow you to configure how your [Git repositories](/docs/cloud/deploy-from-git) get deployed to [Okteto Cloud](/docs/cloud/). You might want to use them if Okteto Cloud cannot detect how to deploy your application from your [deployment manifests](/docs/cloud/deploy-from-git#prerequisites) or simply when you want more control over how your application gets deployed. 
 
+The okteto pipeline is executed relative to the path where the manifest is located. This means, if the manifest is at any subfolder, the pipeline will use the context of the subfolder where the manifest is defined.
```

Comment:
We should also say this for the `dependencies` section when the path to the manifest is defined